app/models/inat_import_job_tracker.rb

```diff
@@ -22,17 +22,25 @@ def status
     import.state
   end
 
-  def elapsed
+  def elapsed_time
     end_time = if status == "Done"
                  ended_at
                else
                  Time.zone.now
                end
-    total_seconds = (end_time - created_at).to_i
-    hours = total_seconds / 3600
-    minutes = (total_seconds % 3600) / 60
-    seconds = total_seconds % 60
-    format("%d:%02d:%02d", hours, minutes, seconds)
+    seconds = (end_time - created_at).to_i
+    time_in_hours_minutes_seconds(seconds)
+  end
+
+  def remaining_time
+    # Can't calculate remaining time until we've imported at least one obs
+    return "" unless imported_count&.positive?
+
+    remaining_importables = importables - imported_count
+    cumulative_avg_import_time = elapsed_seconds / imported_count
+    seconds =
+      (remaining_importables * cumulative_avg_import_time).to_i
```

Comment:
I don't see anything in this calculation that takes into account the time spent on the current import.  I would expect something like:
```
total_expected_time = total_count * avg_import_time
remaining_time = total_expected_time - elapsed_seconds
```
where `avg_import_time` only gets updated after each import completes.